Signal/translations/bin/auto-genstrings

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+TARGETS="Signal/ Pods/SignalServiceKit/ Pods/JSQMessagesViewController/"
+OUTPUT_DIR="Signal/translations/en.lproj/"
+
+if [ ! -d "Signal/src" ]; then
+  echo "Please run this tool from the repository's base directory"
+  exit 1
+fi
+
+mkdir -p "${OUTPUT_DIR}"
+
+find $TARGETS -name "*.m" -print0 | xargs -0 genstrings -a -o "$OUTPUT_DIR"
```

Comment:
I don't think that we want the `-a` as it will just duplicate all the existing strings to the end of the file.